akka-docs/rst/scala/stream/stream-composition.rst

```diff
@@ -248,7 +248,7 @@ The propagation of the individual materialized values from the enclosed modules
 |
 
 To implement the above, first, we create a composite :class:`Source`, where the enclosed :class:`Source` have a
-materialized type of :class:`Promise[Unit]`. By using the combiner function ``Keep.left``, the resulting materialized
+materialized type of :class:`Promise[[Option[Int]]`. By using the combiner function ``Keep.left``, the resulting materialized
```

Comment:
[This](http://doc.akka.io/docs/akka/2.4/scala/stream/stream-composition.html#Materialized_values) refers to [`CompositionDocSpec.scala`](https://github.com/akka/akka/blob/4a7cd147313404eb8137e56163e54f06c23640d0/akka-docs/rst/scala/code/docs/stream/CompositionDocSpec.scala#L181) where the code is as follows.

```
    // Materializes to Promise[Option[Int]]                                   (red)
    val source: Source[Int, Promise[Option[Int]]] = Source.maybe[Int]
```

So `Promise[[Option[Int]]` is correct.